Parsers/ASimAuthentication/Parsers/ASimAuthentication.yaml

```diff
@@ -61,4 +62,5 @@ Parsers:
  - ASimAuthenticationSshd
  - ASimAuthenticationSu
  - ASimAuthenticationVectraXDRAudit
+ - ASimAuthenticationSentinelOne
```

Comment:
the format is not correct in Authentication, pls use below with underscore:
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/54f85ad6-993a-423f-bb73-f8c799128535)

Reply:
Done